jekyll/_style/style-voice.adoc

```diff
@@ -0,0 +1,36 @@
+= Style and voice
+:page-layout: classic-docs
+:icons: font
+:toc: macro
+:toc-title:
+
+== Talk confidently and directly
+Use an active voice, talk directly to the reader using simple, direct, clear and confident language. Using active voice helps us to keep instructions as short as possible, and is, in most cases, the quickest way to convey meaning to the reader
+
+* **Good**: Click **Go** and your project will complete.
+* **Bad**: When you click the **Go** button your project should complete.
+
+== Write for anyone and everyone
+Assume technical competence but explain concepts clearly and simply. Some concepts we write about are complex, and outside a non-technical reader's experience, but the flow of CircleCI docs should be accessible to anyone.
+
+== Word choices
+* Steer clear of making assumptions about the complexity of a process. Avoid descriptions like "easy", "easily" or "simply".
+* Avoid ambiguous language such as "should" and "should be". If something is optional, say so, if not make it clear it is an essential step.
+* When specifying a tool or way of performing an action, use "using" instead of "via". The latter is harder on non-English speakers.
```

Comment:
```suggestion
* When specifying a tool or way of performing an action, use "using" instead of "via". The latter is harder on non-English speakers.
* Avoid contractions, for example "don't", "you're". They make the content harder for non-native english speakers to read.
```